docs/src/model_setup/grids.md

```diff
@@ -34,7 +34,16 @@ grid spacing (Δx, Δy, Δz): (4.0, 4.0, 2.0)
 
 Another crucial keyword is a 3-`Tuple` that specifies the grid's `topology`.
 In each direction the grid may be `Periodic`, `Bounded`, or `Flat`.
-`Flat` dimensions are used to specify two-dimensional and one-dimensional domains.
+`Flat` dimensions are used to specify two-dimensional and one-dimensional domains. 
+Currently only a subset of the possible topologies are available (the limiting factor being the pressure solver):
+
+```
+(Periodic, Periodic, Periodic)
+(Periodic, Periodic, Bounded)
+(Periodic, Bounded, Bounded)
+(Bounded, Bounded, Bounded) (only works on CPUs)
+```
```

Comment:
A small edit could upgrade this block to bona fide julia code:

```julia
topology = (Periodic, Periodic, Periodic)
topology = (Periodic, Periodic, Bounded)
topology = (Periodic, Bounded, Bounded)
topology = (Bounded, Bounded, Bounded)
```

(Note that `topology = (Bounded, Bounded, Bounded)` only works on CPUs.)

I think it'd make the most sense to put this comment _after_ the example that shows how to specify a topology, perhaps?

Reply:
Thanks! I agree that makes it clearer

Reply:
I can take another look if you re-request a review once that's done.